src/shared-components/indicator/index.tsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { COLORS_PROP_TYPES, ThemeColors } from 'src/constants';
 
 import { IndicatorContainer } from './style';
 
 type IndicatorProps = {
   text: string | number;
+  backgroundColor?: ThemeColors;
 };
 
 /**
  * Indicators are used in navigation to help with wayfinding for messages and notifications.
+ *
+ * By adding an optional `backgroundColor` prop, the component can be reused for other purposes
```

Comment:
Since this is an open source library we tend to keep comments more generic than this. Perhaps something added to the previous sentence like `It can also be used for non-navigation purposes for information-intensive pages.` Truly spitballing here 😅

Reply:
oh right my bad, I forgot, will update it to be more generic! 😅